.rhcicd/floorist/floorplan.yaml

```diff
@@ -152,6 +152,25 @@ objects:
             apps.display_name,
             e.org_id,
             nh.status
+      - prefix: insights/notifications/event_types_per_organization
+        query: >-
+          SELECT
+            event_type.display_name::TEXT AS event_type,
+            behavior_group.org_id::TEXT
+          FROM
+            event_type
+          INNER JOIN
+            event_type_behavior AS etb
+              ON etb.event_type_id = event_type.id
+          INNER JOIN
+            behavior_group_action AS bga
+              ON bga.behavior_group_id = etb.behavior_group_id
+          INNER JOIN
+            behavior_group
+              ON behavior_group.id = bga.behavior_group_id
+          GROUP BY
+            event_type.display_name,
+            behavior_group.org_id
```

Comment:
The query does not contain any aggregate function so there's no need for a `GROUP BY` clause.
```suggestion
```

Reply:
True, good catch!